RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernels.cu

```diff
@@ -1,5 +1,132 @@
 #include "RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernelsImpl.h"
 
+/// Compute the number of quadruplet blocks for block size
+inline uint32_t nQuadrupletBlocks(uint32_t blockSize) {
+  // caConstants::maxNumberOfQuadruplets is a constexpr, so the compiler will pre compute the 3*max/4
+  return (3 * caConstants::maxNumberOfQuadruplets / 4 + blockSize - 1) / blockSize;
+}
+
+#define CMS_USE_COOP_GROUPS
+
+#ifndef CMS_USE_COOP_GROUPS
```

Comment:
Would it make sense for this to be more restricted in scope, like `CMS_CA_USE_COOPERATIVE_GROUPS` ?